project-base/storefront/components/PaymentsInOrderSelect/PaymentsInOrderSelect.tsx

```diff
@@ -0,0 +1,123 @@
+import { InfoIcon, SpinnerIcon } from 'components/Basic/Icon/IconsSvg';
+import { Button } from 'components/Forms/Button/Button';
+import { Radiobutton } from 'components/Forms/Radiobutton/Radiobutton';
+import { GoPayGateway } from 'components/Pages/Order/PaymentConfirmation/Gateways/GoPayGateway';
+import { TransportAndPaymentListItem } from 'components/Pages/Order/TransportAndPayment/TransportAndPaymentSelect/TransportAndPaymentListItem';
+import { TransportAndPaymentSelectItemLabel } from 'components/Pages/Order/TransportAndPayment/TransportAndPaymentSelect/TransportAndPaymentSelectItemLabel';
+import { useOrderAvailablePaymentsQueryApi } from 'graphql/generated';
+import { useChangePaymentInOrder } from 'hooks/order/useChangePaymentInOrder';
+import useTranslation from 'next-translate/useTranslation';
+import { useEffect, useState } from 'react';
+import { PaymentTypeEnum } from 'types/payment';
+
+type PaymentsInOrderSelectProps = {
+    orderUuid: string;
+    orderPaymentType: string | undefined;
+    canPaymentBeRepeated: boolean;
+};
+
+export const PaymentsInOrderSelect: FC<PaymentsInOrderSelectProps> = ({
+    orderUuid,
+    canPaymentBeRepeated,
+    orderPaymentType,
```

Comment:
Reading this code, it was unclear to me what `orderPaymentType` actually is (even though it was me who named it like that 😆). Could you rename it to something like `lastUsedOrderPaymentType`?

Reply:
I realized that I don't even need these props here, but I renamed it in the parent component